docs/src/componentDocs/InfoListItem/markdown/InfoListItemExamples.mdx

```diff
@@ -1,180 +1,35 @@
-import Box from '@mui/material/Box';
-import infoListItemExample from '../images/InfoListItemExample.png';
-import infoListItemDense from '../images/InfoListItemDense.png';
-import indicatingStatus from '../images/indicatingStatus.png';
-import infolistItemComponent from '../images/infolistItemComponent.png';
-import infolistItemClickable from '../images/infolistItemClickable.png';
+import {InfoListItem} from '../examples/InfoListItem';
+import {InfoListItemMultiple} from '../examples/InfoListItemMultiple';
+import {InfoListItemStatus} from '../examples/InfoListItemStatus';
+import {InfoListItemContent} from '../examples/InfoListItemContent';
+import {InfoListItemClickable} from '../examples/InfoListItemClickable';
 
 ## Info List Item
 
 An `<InfoListItem>` is a wrapper around the [MUI List Item](https://mui.com/material-ui/api/list-item/) that provides convenient styling and content arrangement options. A basic use case displays a combination of title, subtitle, and icon.
```

Comment:
We are not using `MUI` in react native components, may need to update the description.